ext/yarp/include

```diff
@@ -0,0 +1 @@
+../../include
```

Comment:
Is this symlink and the below necessary?
It causes several editors (Atom, VSCode) to show duplicate matches for global search.